web_src/js/features/repo-issue.js

```diff
@@ -331,6 +331,14 @@ export function initRepoIssueReferenceRepositorySearch() {
     });
 }
 
+export function initRepoIssueTitleFocus() {
+  const issueTitle = document.getElementById('issue_title');
```

Comment:
Instead of hard-coding `issue_title` here, I would suggest to introduce a general CSS selector:

```
<input type="js-auto-focus-end ...">
```

Reply:
I wonder if we could keep the DOM `autofocus` property, detect when it runs and then just set the cursor to the right position when a class like `js-autofocus-end` is present. That should in theory also work for content appended to DOM via JS.

Reply:
Not sure if it's possible to detect autoFocus in JS. One could install a `focus` event listener on `document.body` but the problem will be telling apart a autofocus from a regular user interaction.

Reply:
Yeah no I think trying to detect autofocus is a hopeless effort. Would suggest to follow @wxiaoguang and add this simple class that will manually focus the first matching element on page load.

Reply:
Updated